### PR TITLE
Elaborate on purpose in README

### LINKED ISSUE #76: README should be more clear on what this repo actually is.
A shim for what? what is the purpose, what is the end goal, where is it being used currently etc.

### PR DESCRIPTION
**Pull request by BrianSipple**
Closes #76 😄 .

### CODE REVIEW

**Comment by stefanpenner** [comment]
cc @locks / @acorncom

**Comment by locks** [comment]
@stefanpenner 👍 

Do we need to update which package managers this repository supports? I see something about jspm inside `package.json`, and we have a `component.json` as well.

**Comment by BrianSipple** [comment]
I've used [Ember through jspm](https://github.com/jspm/registry/blob/master/registry.json#L282) on a number of occasions, and Composer definitely seems to be on the list [for our other packages](https://github.com/components/ember-data#package-managers).

I can update the PR with both if that would be appropriate.

File: README.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
 Ember.js
 ========
 
-Shim repository for [Ember Application Framework](http://emberjs.com/).
+_Shim repository for [Ember Application Framework](http://emberjs.com/)._
 
 This package provides the core of the ember.js framework.
+When a new [build is released](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html),
+package managers will find it here.
```
Comment (rwjblue) [inline README.md]: I am not sure how to word it, but we should make it clear that this repo is for usage with package managers that do not have "their own storage".  Specifically speaking, we do not plan to use this repo when we publish and consume Ember from NPM (#soon).

Can you think of a good way to say that?

Comment (rwjblue) [inline README.md]: Sounds great!

Comment (BrianSipple) [inline README.md]: @rwjblue Just updated and rebased. I also added JSPM and Component to the list, as mentioned above.

**Pull request merged**